src/rust/include/rust/orchard/wallet.h

```diff
@@ -287,6 +291,29 @@ OrchardSpendInfoPtr* orchard_wallet_get_spend_info(
         const unsigned char txid[32],
         uint32_t action_idx);
 
+/**
+ * Run the garbage collection operation on the wallet's note commitment
+ * tree.
+ */
+bool orchard_wallet_gc_note_commitment_tree(OrchardWalletPtr* wallet);
+
+/**
+ * Write the wallet's note commitment tree to the provided stream.
```

Comment:
I presume this returns `false` on I/O error?

Reply:
Yes.